### PR TITLE
Migration catalog e2e

### PR DESCRIPTION
**Pull request by Hugo-Hontans**
Fix catalog app and catalog e2e tests

### CODE REVIEW

File: apps/catalog/catalog-e2e/src/support/pages/marketplace/ViewPage.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export default class ViewPage extends NavbarPage {
 
   public clickWishListButton() {
     cy.get('.heart-button').click();
+    cy.wait(2000);
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog-e2e/src/support/pages/marketplace/ViewPage.ts]: Why do you have to wait ?

File: apps/catalog/catalog/src/app/marketplace/movie/search/search.component.ts [hunk 2]
```diff
@@ -264,6 +264,7 @@ export class MarketplaceSearchComponent implements OnInit {
   }
 
   public goToMovieDetails(id: string) {
+    console.log('coucou')
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/marketplace/movie/search/search.component.ts]: I think you can remve that ^^

File: apps/catalog/catalog/src/app/marketplace/movie/search/search.component.ts [hunk 3]
```diff
@@ -264,6 +264,7 @@ export class MarketplaceSearchComponent implements OnInit {
...
     this.router.navigateByUrl(`c/o/catalog/${id}`);
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/marketplace/movie/search/search.component.ts]: Is it not `c/o/marketplace/${id}` instead ?

File: libs/organization/src/lib/cart/+state/cart.service.ts [hunk 4]
```diff
@@ -137,9 +137,11 @@ export class CartService extends CollectionService<CartState> {
     try {
       const wishlist = orgState.wishlist.map(wish => {
         if (wish.status === 'pending') {
-          wish.movieIds = wish.movieIds.filter(id => id !== movieId)
+          const movieIds = wish.movieIds.filter(id => id !== movieId);
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/cart/+state/cart.service.ts]: Even better : 
```typescript
return {
  ...wish,
  movieIds: wish.movieIds.filter(id => id !== movieId)
}
```

File: apps/catalog/catalog-e2e/src/support/pages/marketplace/ViewPage.ts (hunk 1 above)
Comment (Hugo-Hontans) [reply apps/catalog/catalog-e2e/src/support/pages/marketplace/ViewPage.ts]: We check when adding a film if the counter on the heart icon increases and cypress goes too fast if we don't wait. I dow it to 500

**Pull request merged**